docs/data/toolpad/core/components/use-notifications/ScopedNotification.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {
+  NotificationsProvider,
+  useNotifications,
+} from '@toolpad/core/useNotifications';
+import Button from '@mui/material/Button';
+import { Box, Snackbar, styled } from '@mui/material';
+
+const notificationsProviderSlots = {
+  snackbar: styled(Snackbar)({ position: 'absolute' }),
+};
+
+function ScopedContent() {
+  const notifications = useNotifications();
+  return (
+    <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+      <Button
+        onClick={() => {
+          notifications.show('Consider yourself notified!', {
+            autoHideDuration: 3000,
+          });
+        }}
+      >
+        Notify me
+      </Button>
+    </Box>
+  );
+}
+
+export default function ScopedNotification() {
+  return (
+    <Box sx={{ width: '100%', height: 150, position: 'relative' }}>
+      <NotificationsProvider slots={notificationsProviderSlots}>
```

Comment:
Another possibility could be to pass an optional target to render on when calling `notifications.show` instead of using two providers (which I guess kind of nullifies the top-level `NotificationsProvider`)

Reply:
I'm not sure how that would work, the provider takes care of queueing and deduplicating multiple concurrent notifications.